cmd/tools/cloudburst/cloudburst.go

```diff
@@ -397,6 +410,55 @@ func (b *Blaster) createInstallation() (*cloud.Installation, error) {
 	return installationDTO.Installation, nil
 }
 
+func isValidInput(database, filestore, installSize string) error {
+	databaseCheck := func() error {
+		for _, validDB := range []string{
+			cloud.InstallationDatabaseMysqlOperator,
+			cloud.InstallationDatabaseSingleTenantRDSMySQL,
+			cloud.InstallationDatabaseSingleTenantRDSPostgres,
+			cloud.InstallationDatabaseMultiTenantRDSMySQL,
+			cloud.InstallationDatabaseMultiTenantRDSPostgres,
+		} {
+			if validDB == database {
+				return nil
+			}
+		}
+		return errors.Errorf("invalid database requested: unknown database type %s", database)
+	}
+
+	filestoreCheck := func() error {
+		for _, validFilestore := range []string{
```

Comment:
We have an `IsValid()` check for filestores and databases in the cloud model. Using those would clean this up a bit.

```
// IsSupportedFilestore returns true if the given filestore string is supported.
func IsSupportedFilestore(filestore string) bool {
	return filestore == InstallationFilestoreMinioOperator ||
<CUT>
```

Reply:
Thanks @gabrieljackson I looked for such a function, but obviously not hard enough :P